src/App.tsx

```diff
@@ -1,12 +1,54 @@
 import React from 'react';
 import './App.scss';
+import 'bulma';
+import { Table } from './components/Table';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+import { getAll, get5First, getRedGoods } from './api/goods';
 
-const App: React.FC = () => (
-  <h1>Dynamic list of Goods</h1>
-);
+type State = {
+  goods: Good[]
+};
+
+class App extends React.Component<{}, State> {
+  state = {
+    goods: [],
+  };
+
+  componentDidMount() {
+    this.loadAll();
+  }
+
+  loadAll = () => {
+    getAll()
+      .then(goods => {
+        this.setState({ goods });
+      });
+  };
+
+  load5First = () => {
+    get5First()
+      .then(goods => {
+        this.setState({ goods });
+      });
+  };
+
+  loadRedGoods = () => {
+    getRedGoods()
+      .then(goods => {
+        this.setState({ goods });
+      });
+  };
```

Comment:
create one function that accepts callback (getAll, load5First or loadRedGoods) and calls it and write the result to state